void-gr-filters.txt

```diff
@@ -42,6 +42,9 @@ capital.gr/banners/StaticBanners/*
 cdn1.telemetryverification.net/tv2n/presenter/deploys/inbanner/*
 connextra.com/VistaBet/*
 contra.gr/XML/sponsors/
+corfuland.gr###first-big-banner
+corfuland.gr##.module-shadow
```

Comment:
can you add these 2 under `!Element-Hiding Rules` please ?

Reply:
Done!